openstack/imageservice/v2/imageimport/requests.go

```diff
@@ -0,0 +1,20 @@
+package imageimport
+
+import "github.com/gophercloud/gophercloud"
+
+// ImportMethod represents valid Import API method.
+type ImportMethod string
+
+const (
+	// GlanceDirectMethod represents glance-direct Import API method.
+	GlanceDirectMethod ImportMethod = "glance-direct"
+
+	// WebDownloadMethod represents web-download Import API method.
+	WebDownloadMethod ImportMethod = "web-download"
+)
+
+// Get retrieves Import API information data.
+func Get(c *gophercloud.ServiceClient) (r GetResult) {
```

Comment:
Since this API call isn't returning anything about an import task, should we rename this to `GetImportMethods`? It might make it more clear to users.

Reply:
Given this might return more than just import methods in the future, `GetImportMethods` is probably not a good name.

I guess since this is the only `GET` method under `imageimports` at this time, we can keep this as `Get`. That feels a little strange, but I guess that's how it is :)